src/driver/voodoo.ml

```diff
@@ -259,24 +277,66 @@ let of_voodoo pkg_name ~blessed =
       in
       let packages = List.filter_map (fun x -> x) (last :: packages) in
       let packages = List.map process_package packages in
+      let pkg = List.hd packages in
+      Logs.debug (fun m -> m "Package: %a\n%!" Packages.pp pkg);
```

Comment:
Is the first package always the "main" one? Maybe we should just print`pkg_name` here